x-pack/metricbeat/module/statsd/server/data_test.go

```diff
@@ -194,7 +194,7 @@ func (u *testUDPEvent) GetMeta() server.Meta {
 	return u.meta
 }
 
-func process(t testing.TB, packets []string, ms *MetricSet) {
+func process(t *testing.T, packets []string, ms *MetricSet) error {
```

Comment:
Was there any reason to change it from `testing.TB` to `*testing.T`? I think it is good to keep these test helpers generic when possible.

On the other hand with current code it is not using `t`, so it seems that you can just remove the argument.
```suggestion
func process(packets []string, ms *MetricSet) error {
```

Reply:
of course, thanks!